actionpack/lib/abstract_controller/base.rb

```diff
@@ -150,6 +150,13 @@ def available_action?(action_name)
       _find_action_name(action_name)
     end
 
+    # Tests if a response body is set. Used to determine if the
```

Comment:
This should document the intention of the method. like just `Tests if a response body is set`. It doesn't matter how it was implemented or where it is used internally.